src/data_storage/export/export_storage.ts

```diff
@@ -41,6 +43,13 @@ export default class ExportStorage extends PostgresStorage{
                             return
                         }
 
+                        const event: EventT = {
```

Comment:
I would add an event on the SetCompleted function as well.

Reply:
Should the existing event be moved to SetCompleted? If we have two events in this class, do we need another event type?

Reply:
Move to SetCompleted